examples/main.tf

```diff
@@ -1,6 +1,6 @@
 terraform {
   required_providers {
-    port = {
+    port-labs = {
       source  = "port-labs/port-labs"
       version = "~> 0.2.0"
```

Comment:
The version is suuuuper wrong

Reply:
Also the example does not include any where specifying the client ID and secret, it should be added for a complete working exmaple